share/sequelize/requireModels.js

```diff
@@ -13,7 +13,7 @@ const readModelFilenamesFromPath = (dirPath) => (
 
 
 module.exports = ({ sequelize, Sequelize, modelsPath }) => {
-  const mPath = modelsPath || path.resolve(__dirname, './models')
+  const mPath = modelsPath || path.resolve(__dirname, '.././models')
```

Comment:
`'../models')`, although it should still resolve

Reply:
facepalm - should've caught it myself while tweaking the paths a hundred times :-)